app/views/kitten/components/buttons/button-tag.html.erb

```diff
@@ -1,13 +1,12 @@
 <div class="karl-Example__column">
   <%= example "Hydrogen tag button" do %>
-    <button class="k-ButtonTag k-ButtonTag--hydrogen">
-      Tag
-    </button>
+    <%= react_component('ButtonTag') %>
   <% end %>
 
   <%= example "Selected tag button" do %>
-    <button class="k-ButtonTag k-ButtonTag--hydrogen is-selected">
-      Tag
-    </button>
+    <%= react_component('ButtonTag', props: {
+        isSelected: true,
```

Comment:
Dans des PR récentes, on a pris l'habitude de ne pas mettre le `is`. Du coup on aurait que `selected: true`.